src/components/Card/Card.tsx

```diff
@@ -166,13 +166,19 @@ const CardWrapper: React.FC<any> = chakra(
           id={id}
           className={className}
           ref={ref}
-          sx={styles}
+          sx={styles.base}
```

Comment:
Ah, here and on line 179, you don't need to do `styles.base` because `styles.base` is already being passed to the `<CardWrapper` component on line 321. So in this case you should only need to do `sx={styles}`. Same on line 179.

Reply:
Wow, that seemed to do the trick.  Thanks.